flask_bootstrap/__init__.py

```diff
@@ -84,26 +88,38 @@ def load_css(version=VERSION_BOOTSTRAP):
         serve_local = current_app.config['BOOTSTRAP_SERVE_LOCAL']
         bootswatch_theme = current_app.config['BOOTSTRAP_BOOTSWATCH_THEME']
 
+        if version == VERSION_BOOTSTRAP and serve_local is False and bootstrap_sri is None:
+            bootstrap_sri = BOOTSTRAP_CSS_INTEGRITY
+
         if not bootswatch_theme:
             base_path = 'css/'
         else:
             base_path = 'css/swatch/%s/' % bootswatch_theme.lower()
 
         if serve_local:
-            css = '<link rel="stylesheet" href="%s" type="text/css">' % \
-                url_for('bootstrap.static', filename=base_path + css_filename)
+            if bootstrap_sri:
+                css = ('<link rel="stylesheet" href="%s" integrity="%s" crossorigin="anonymous">' %
+                       (url_for('bootstrap.static', filename=base_path + css_filename), bootstrap_sri))
+            else:
+                css = ('<link rel="stylesheet" href="%s">' %
+                       url_for('bootstrap.static', filename=base_path + css_filename))
         else:
             if not bootswatch_theme:
-                css = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@%s/dist/css/%s"' \
-                    ' type="text/css">' % (version, css_filename)
+                if bootstrap_sri:
+                    css = ('<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@%s/dist/css/%s"'
+                           ' integrity="%s" crossorigin="anonymous">' % (version, css_filename, bootstrap_sri))
+                else:
+                    css = ('<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@%s/dist/css/%s">' %
+                           (version, css_filename))
             else:
-                css = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@%s/dist/%s/%s"' \
-                    ' type="text/css">' % (version, bootswatch_theme.lower(), css_filename)
+                css = ('<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@%s/dist/%s/%s">' %
+                       (version, bootswatch_theme.lower(), css_filename))
         return Markup(css)
 
-    @staticmethod
+    @staticmethod  # noqa: C901
```

Comment:
```
flask_bootstrap/__init__.py:120:5: C901 'Bootstrap.load_js' is too complex (15)
```

The inline ignore should be placed on line 120

Reply:
I saw that, it shows line 119 on my PC before, so I add the ignoration to line 119 and then it passed the test.
The inconsistency between my PC and the CI runner is kind of odd, will try to find it out later.

Reply:
I moved the ignoration from line 119 to 120, then it failed the test on my PC...
@greyli any suggestions?
```
(venv) bootstrap-flask $tox -e flake8                                                                                                            sri*
flake8 installed: flake8==3.9.2,importlib-metadata==4.5.0,mccabe==0.6.1,pycodestyle==2.7.0,pyflakes==2.3.1,typing-extensions==3.10.0.0,zipp==3.4.1
flake8 run-test-pre: PYTHONHASHSEED='304898279'
flake8 run-test: commands[0] | flake8 flask_bootstrap tests
flask_bootstrap/__init__.py:119:5: C901 'Bootstrap.load_js' is too complex (15)
ERROR: InvocationError for command /home/yuxiaoy/python/oss/bootstrap-flask/.tox/flake8/bin/flake8 flask_bootstrap tests (exited with code 1)
______________________________________________________________________ summary ______________________________________________________________________
ERROR:   flake8: commands failed
```

Reply:
Not sure what happened here, I just move the ignore comment to L120...